llvm/test/lab2/sharapov_georgiy/test.ll

```diff
@@ -0,0 +1,190 @@
+; RUN: opt -load-pass-plugin=%llvmshlibdir/LoopFramer%shlibext -passes=LoopFramer -S %s | FileCheck %s
```

Comment:
Could you please also provide a test for a function with just `if-else` condition to check that no guards is added?